### DISCUSSION TITLE
Specify how to save custom dictionary in txt file in device

### DISCUSSION THREAD

**Discussion by Nireno146** [tadashi-aikawa/obsidian-various-complements-plugin#301] [Q&A] [answered]
![20240612_000614.jpg](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/171968208/d7189379-8ac0-4ba3-9b02-ad489cc76536)

**Comment by tadashi-aikawa** [comment, marked as answer]
@Nireno146 
> Specify how to save custom dictionary in txt file in device

The title mentions a txf file, but it looks like an md file is specified in the screenshot. Are the settings correct?

**Reply by AhmadFaizi01** [reply]
Hello, I have the same issue as @Nireno146. I have verified the path multiple times, but I can't change the permissions of the directory because it is on a USB key
![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/130755459/3c42d0fc-781b-418a-9aa1-07158975e4b1)
this the path of my dictionary 
``D:\PortableApps\ObsidainPortable\Obsidian_Notes\Dictionary``
![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/130755459/f626c771-b815-4db7-a11e-78ef1481555c)

**Reply by tadashi-aikawa** [reply]
@AhmadFaizi01 
It doesn't support absolute path.

https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/4.%20Options/4.6.%20Custom%20dictionary%20complement/%E2%9A%99%EF%B8%8F%20Custom%20dictionary%20paths/

**Reply by AhmadFaizi01** [reply]
ok got it thanks for your response. i fixed it now it works.
